### PR TITLE
release: cut the 6.0.0 release

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

**Comment by SvetoslavTsenov** [comment]
test --ignore

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+
+* Old behavior:
+ - iOS/Android:
+    - double tap: child tap -> parent tap -> child double tap -> parent double tap
+    - tap: child tap -> parent tap
+
+New behavior:
+ - iOS
+    - double tap: child double tap
+    - tap: child tap
```
Comment (MCurran16) [inline CHANGELOG.md]: @SvetoslavTsenov This new behavior is in correct as it was reverted for iOS.

See https://github.com/NativeScript/NativeScript/commit/8722075

**Comment by SvetoslavTsenov** [comment]
test package_version@rc

File: CHANGELOG.md (hunk 1 above)
Comment (SvetoslavTsenov) [reply CHANGELOG.md]: Thanks, @MCurran16
Ping @manoldonev

Comment (manoldonev) [reply CHANGELOG.md]: I do not believe https://github.com/NativeScript/NativeScript/commit/8722075 cancels the fix for contradictory gesture events -- as far as I tested with the latest version of the codebase we are not raising tap & double tap at the same time.

**Comment by manoldonev** [comment]
test package_version@rc

**Comment by manoldonev** [comment]
test --ignore

**Comment by SvetoslavTsenov** [comment]
test --ignore

**Pull request merged**